app/components/T/index.tsx

```diff
@@ -5,15 +5,15 @@
  */
 
 import React, { memo } from 'react';
-import styled, { FlattenSimpleInterpolation } from 'styled-components';
```

Comment:
Dont we need the FlattenSimpleInterpolation ?

Reply:
Using CSSInterpolation from '@emotion/serialize';